### PR TITLE
JIP module port

### PR DESCRIPTION
**Pull request by EM-Creations**
Ported.

### CODE REVIEW

File: modules/jip/functions/JIP/fn_JIP_ClientInit.sqf [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+
+#include "..\..\settings.sqf"
+
+if (hasInterface && {(str (side player)) != "LOGIC"}) then {
```
Comment (PiZZAD0X) [inline modules/jip/functions/JIP/fn_JIP_ClientInit.sqf:5]: can use an exitwith instead. !hasInterface will detect server + clients. `player` wont exist on the headless client/server.

File: modules/jip/teleportAction.sqf [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+
+if !((_target call FUNC(isAlive)) && {(!(INVEHICLE(_target)) || {((vehicle _target) call FUNC(getEmptyPositions)) isEqualTo []})}) then {
 
 	private _rank = -1;
 	private _count = 0;
-    
+
     (units group player) select {
         _x call FUNC(isAlive) &&
 		{!(_x isEqualTo player)}
```
Comment (PiZZAD0X) [inline modules/jip/teleportAction.sqf:12]:
```suggestion
		{_x isNotEqualTo player}
```

Comment (EM-Creations) [reply modules/jip/teleportAction.sqf]: Fixed.

File: modules/jip/functions/JIP/fn_JIP_ClientInit.sqf (hunk 1 above)
Comment (EM-Creations) [reply modules/jip/functions/JIP/fn_JIP_ClientInit.sqf]: Fixed.

**Review by PiZZAD0X — APPROVED** [review]

**Pull request merged**